cpp/vm/evmzero/interpreter.cc

```diff
@@ -986,8 +986,8 @@ struct Impl<OpCode::BLOBHASH> {
 
   static OpResult Run(uint256_t* top, Context& ctx) noexcept {
     const auto tx_context = ctx.host->get_tx_context();
-    if (top[0] < tx_context.blob_hashes_count) {
-      top[0] = ToUint256(tx_context.blob_hashes[static_cast<int64_t>(top[0])]);
+    if (static_cast<size_t>(top[0]) < tx_context.blob_hashes_count) {
```

Comment:
I think this change is eliminating an overflow protection.

In the old version, the `top[0]` value was a `uint256_t` while the `tx_context.blob_hash_count` is a `size_t`. In the implementation of the `operator<` of the `uint256_t` the `size_t` value is extended to a 256-bit value and then compared.

Now, in the new version, the `uint256_t` value is converted into a `size_t`, ignoring overflows, and the compared to the number of hashes. I guess this is not the intention, is it?